README.md

```diff
@@ -1,3 +1,13 @@
-# DIDSign
+# DIDsign
 
-The source code of https://didsign.io/
+DIDsign is a website that allows you to:
+
+* sign electronic documents with your KILT on-chain DID directly in your browser,
+* insert credentials as a part of the signature for these documents,
+* use timestamping to proof the approximate time you signed them,
+* verify documents that have been signed with a KILT on-chain DID,
+* display additional information about the signers on-chain DID, such as e.g. the on-chain DID, service endpoints that are associated with this on-chain DID and KILT credentials that have been made available via the service endpoints.
```

Comment:
Since it is always one signer's signature that we are talking about:
```suggestion
* display additional information about the signer's on-chain DID, such as e.g. the on-chain DID, service endpoints that are associated with this on-chain DID and KILT credentials that have been made available via the service endpoints.
```